.gitignore

```diff
@@ -6,7 +6,9 @@ dump.rdb
 
 db.sqlite3
 .venv
-static
```

Comment:
I needed to have the static files in the repo and I'm not sure why there were not tracked. I did track them. Please let me know if you feel uncomfortable with that.